bigbluebutton-html5/imports/ui/components/video-provider/many-users-notify/container.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { withTracker } from 'meteor/react-meteor-data';
+import Meetings from '/imports/api/meetings/';
+import Auth from '/imports/ui/services/auth';
+import Users from '/imports/api/users/';
+import LockViewersService from '/imports/ui/components/lock-viewers/service';
+import ManyUsersComponent from './component';
+
+export default withTracker(props => ({
+  viewersInWebcam: Users.find({
+    hasStream: true,
```

Comment:
I'd consider also adding the meetingId in the selector to avoid odd edge cases like for moderators during breakouts

Reply:
Okay